README.md

```diff
@@ -141,8 +141,8 @@ for now.
 
 Ensure you have the latest LTS version of Node.js installed.
 
-Using `yarn` instead of `npm` is recommended. Please see the Yarn 1 [install
-guide](https://classic.yarnpkg.com/docs/install) if you do not have it
+Using `yarn` instead of `npm` is recommended. Please see the Yarn 1 [`install
+guide`](https://classic.yarnpkg.com/docs/install) if you do not have it
```

Comment:
See other comment

```suggestion
Using `yarn` instead of `npm` is recommended. Please see the Yarn 1 [install
guide](https://classic.yarnpkg.com/docs/install) if you do not have it
```